README.md

```diff
@@ -185,6 +189,8 @@ bundle exec cap prod deploy  # prod
 # Note: there is no QA
 ```
 
+By default, deployment will use the deployed branch's code to build a fresh Docker image locally, and will restart `docker compose`, which will cause it to pick up the new image.  If you'd rather use an image from the Docker Hub registry, see comment by the `build` field in the `x-airflow-common` section of `compose.prod.yaml` for more detailed instructions.
```

Comment:
a bit more explanation since we don't have consistent patterns around this, since we don't often deploy dockerized apps to our on prem VMs.